src/internal.c

```diff
@@ -11933,8 +11933,9 @@ static int ChachaAEADDecrypt(WOLFSSL* ssl, byte* plain, const byte* input,
 
 #if defined(BUILD_AESGCM) || defined(HAVE_AESCCM)
 
-#if (!defined(HAVE_FIPS) && !defined(HAVE_SELFTEST)) || \
-    (defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION >= 2))
+#if !defined(WOLFSSL_TI_CRYPT) && \
```

Comment:
You'll also need to set this additional macro logic at lines 12060, 12212 and 14167. Also in keys.c 2614 and 2638. Then the `wc_AesGcmSetIV` stub below will not be required and this change would be okay.